src/utils/truncateWalletAddress.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Truncates a wallet's address
+ */
+const truncateWalletAddress = (address: string, startLength = 4, endLength = 4) => {
+  return `${address.substring(0, startLength)}...${address.substring(address.length - endLength)}`
```

Comment:
Noice. If you do a search for `accountEllipsis` you'll see a few bits in the codebase that could use this. Any chance you could find & replace and remove that helper?

Reply:
Ha, weird...I must not have pushed my changes. I did update that.